_search-plugins/knn/knn-index.md

```diff
@@ -64,18 +68,21 @@ SIMD optimization is applicable only if the vector dimension is a multiple of 8.
 
 For x64 architecture, the following versions of the Faiss library are built and shipped with the artifact:
 
-- `libopensearchknn_faiss.so`: The non-optimized Faiss library without SIMD instructions. 
+- `libopensearchknn_faiss_avx512_spr.so`: The Faiss library containing advanced AVX512 SIMD instructions for Intel® Sapphire Rapids or a newer-generation processor.
```

Comment:
newer-generation processor, available on public clouds such as AWS on c/m/r 7i or newer instances.

Reply:
not sure if we can mention about AWS on opensource documentation, @kolchfa-aws do you know ?

Instead shall we mention the CPU flags like "newer-generation processor with CPU flags avx512_fp16, avx512_bf16, avx512_vpopcntdq" ?

Reply:
most customers reading this doc will try opensearch and its features on some public cloud. They're unlikely to understand intel product family names like 'Sapphire Rapids' etc. Hence the c/m/r7i would be helpful to them to pick the right product I thought. 

I think its better to skip the CPU flag details. It's very low level system details, and it'll create more confusion to a reader of opensearch.

Reply:
got it, will update. Thanks!